Directory.Build.props

```diff
@@ -10,6 +10,7 @@
     <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
     <VersionSuffix Condition="'$(VersionSuffix)'!='' AND '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>latest</LangVersion>
```

Comment:
/cc @Eilon how do you feel about using the latest C# everywhere? Maybe this should be latest minor since we don't want to use C# 8 until it's done

Reply:
@davidfowl The settings go the other way. Latest major (`default`) is like LTS and latest minor (`latest`) is like FTS.  However, we could explicitly say `7.1`.

Reply:
Does `latest` mean "Latest RTM version" or "actual latest experimental crazy town insane features"?